pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.epam</groupId>
     <artifactId>parso</artifactId>
-    <version>2.0.2</version>
+    <version>2.0.3</version>
```

Comment:
We haven't yet released version 2.0.2 to maven repo, so this changes can be added to version 2.0.2.